Exception on getting blog avatar

To get the blog name and avatar I am doing following:

```
`for (Blog blog : user.getBlogs()) {
           String blog_name = blog.getName();
            Log.i("LOGIN_TUMBLR", "blog_name: "+blog_name);
            String blog_avatar = blog.avatar();
            Log.i("LOGIN_TUMBLR", "blog_avatar" +
                    ": " + blog_avatar);

    }`
```
 
Output: 

` Caused by: com.tumblr.jumblr.exceptions.JumblrException: �PNG
                                                                     
                                                                     ������IHDR������@������@������%�������	pHYs������H������H��F�k>����$IDATh�MzَdIr�m��ވȈ\k_�z����&GHI� =�$�?��>B$@ ��8"%n�=������ʪ�\b����L7�@O �ss[�9f�o�ݿQU�.'��f��HDb�"����H���Q�*��03B��X��(�i)EUU���������ϥ��ms����u]��������J�nUڮM]����m�֧g��j�)
                                                                     " �����<w��,Xq���x�L����� ������;"�" ���#���u])�������z[۶UU3K9��̭����۬��Z��RZͪ:[���u�T����d�Df6Gw�� 7DB7F"$�:��B����N́����p@�D�|��R�߽�{�RJ)9g3334t�Rr�e�n�v�J�,�%gI���n�n׭�!s�n�憆D$�e��DH��d�B��wW��j�������������]�����������M����;����R�MU�z��a�eۭ�u��n�ۙ%U+�ri90��#`������s*E$F4��3��"����H�u)���DD����s���@��G�w����r�	����U��2_-f��]����Y}YҲk��;7�������������F}�q[�E�ބ�T5�LD�H�}$�#b���b3���uݮS���sD"��b�v�n���b�^��ߗU�EZ�1m4ꔳ�! !"���_@�*�"�����޸�����������"��j�s���sN]J��+���![�.w]��u���K��K�U��:��3K%����IPn8�b�073g'CP3tG")E���c"BĽ����Oo�#b���������kι��������pQw/I��E^�R�P�����EW�c)J"U�9	#��Q�s�97C�(*�"	�j�f@""-���HDz�����j�u��80'G7K)����R�r�9��J��������jɹ�.M���`�i{����,��W�DNH�䀮��cS��I�����U���	"8�#��"⛬p�EM��	��N�g�VI)��rJm�%70S5����23���UM���|�,��h"ǋ���ˬ�D� U�lG62H��0lp0�:n�F���.[��NH�Ȁ��n����P������-O�tů\�d)��KQ��tp7wS7wwPtpU3-%'�\J9:;m���[׆a��ZН�9��f���W��vw��Pdg$"B�o��/�	���l72�#3������r���t�\�e�2f)9��a��4
                                                                     �y�`_�ܝ�C�Fb�l��r3-i�v��13!�;b�X�~eH�n�Z��h��4����͙��BER���YXs�h���]�F�(�=5�Q�������/��ԓ�����2�"�C������0q4�.�N����z�x���Ȏ�������0�Nsp��@HQG����X�Y��g��l>3U4��,y$��ǃ�*.�
                                                                     }G�
                                                                     ���Ш?3-jn����L�'eV��g/�?y>_-�ͥ���@�/���w-JL����`Ű���h�T��A�A�bɌ�|�\̦33CDG��8���E���� !q��	��92�;�Q�А�&R+Y}��t1{v�(��*��KR��=�Y8��ݽ�
                                                                     ������Z2dl4�'�d�� �(�bq��L(
                                                                     K�uV��|���mm���KT���0�y)I��ՌZ4�l����<�Ki0�2Om��J�iK束M�.��ꪊ������2��#����678���8�&Uc	��XEQ+`8[Ζ�y+B1
`

I am getting this exception when I try to get the avatar of the blog. There were no problems getting the name. Thanks.

--- a/src/main/java/com/tumblr/jumblr/request/RequestBuilder.java
+++ b/src/main/java/com/tumblr/jumblr/request/RequestBuilder.java
@@ -44,7 +44,7 @@ public class RequestBuilder {
         HttpURLConnection.setFollowRedirects(false);
         Response response = request.send();
         HttpURLConnection.setFollowRedirects(presetVal);
-        if (response.getCode() == 301) {
+        if (response.getCode() == 301 || response.getCode() == 302) {
             return response.getHeader("Location");
         } else {
             throw new JumblrException(response);

